hugo/content/3_how-tos/31_account-management/312_managing-user-groups.md

```diff
@@ -43,18 +43,20 @@ Following are the features of a user group:
 
 To create/delete a user group:
 
-1. In the rapyuta.io console, click **Account > Organization**. 
+1. In the rapyuta.io console, click **Account > Organization**.
 2. Click the **User Groups** tab.
 3. To create a user group:
-  * Click **Create User Group**, and enter:
-    |Field|Description|
-    |-----|-----------|
-    |**Group Name**| Enter the name of the group.|
-    |**Group Description**| Enter the description. |
-    |**Projects**| Click to link projects to the group.|
-    |**Members**| Click to link group members.|
-  * Click **Submit**.
-4. To leave a group, click the exit icon below the group. On leaving, you will lose access to all the projects linked to the group. 
+    * Click **Create User Group**, and enter:
+
+      |Field|Description|
+      |-----|-----------|
+      |**Group Name**| Enter the name of the group.|
+      |**Group Description**| Enter the description. |
+      |**Projects**| Click to link projects to the group. Additionally, you can choose their role by selecting either **Viewer** or **Admin** from the dropdown menu.|
+      |**Members**| Click to link group members. Additionally, you have the option to designate the user as an admin. |
```

Comment:
**Suggestion**: to designate the user as **Group Admin.**

With just admin, it might be confusing with the role in the project